build/images/ipfixcollector/README.md

```diff
@@ -0,0 +1,17 @@
+# images/ipfixcollector
+
+This Docker image is based on Ubuntu 18.04 which includes ipfix collector based on libipfix, a C library.
+In this image, IPFIX collector listening on tcp:4739 port.
+
+libipfix package is downloaded from https://svwh.dl.sourceforge.net/project/libipfix/libipfix/libipfix-impd4e_110224.tgz
```

Comment:
rather than this line which doesn't bring much, you can use a markdown link to http://libipfix.sourceforge.net/ when you mention libipfix above.

Reply:
done